spec/components/admin/participants/table_spec.rb

```diff
@@ -4,13 +4,13 @@
   let!(:participant_profiles) { create_list :participant_profile, rand(11..15) }
   let(:page) { rand 1..2 }
 
-  component { described_class.new profiles: ParticipantProfile.all, page: page }
+  component { described_class.new profiles: ParticipantProfile.all.order(:id), page: page }
   request_path "/admin/participants"
 
   stub_component Admin::Participants::TableRow
 
   it "renders table row for each participant profile on this page" do
-    expected_profiles = participant_profiles.each_slice(10).to_a[page - 1]
+    expected_profiles = participant_profiles.sort_by(&:id).each_slice(10).to_a[page - 1]
```

Comment:
what were they ordered by before? :O

Reply:
They weren't, which was causing issues on pagination. This component is expecting olready ordered relation.

Reply:
Depends which way the wind was blowing 🤷